test/remote-method.test.js

```diff
@@ -56,6 +56,32 @@ describe('loopback:remote-method generator', function() {
     });
   });
 
+  it('adds an entry to common/models/{name}.json with nonStatic method',
+  function(done) {
+    var methodGenerator = givenMethodGenerator();
+    helpers.mockPrompt(methodGenerator, {
+      model: 'Car',
+      methodName: 'prototype.myRemote',
+      desription: 'This is my first remote method',
+      httpPath: '',
+      acceptsArg: '',
+      returnsArg: ''
+    });
+
+    methodGenerator.run(function() {
+      var definition = common.readJsonSync('common/models/car.json');
+      var methods = definition.methods || {};
+      expect(methods).to.have.property('prototype.myRemote');
+      expect(methods['prototype.myRemote']).to.eql({
+        isStatic: false,
```

Comment:
I find this weird, if your test is assuming static/prototype flag is encoded in the name, then why is there `isStatic` property set too?